esque/cli/commands.py

```diff
@@ -265,20 +261,18 @@ def describe_broker(state, broker_id):
 
 @describe.command("consumergroup")
 @click.argument("consumer-id", required=False)
-@click.option("-v", "--verbose", help="More detailed information.", default=False, is_flag=True)
+@click.option("--raw-output", help="Get raw output", default=False, is_flag=True)
```

Comment:
```suggestion
@click.option("--all-partitions", help="List status for all topic partitions instead of just summarizing each topic.", default=False, is_flag=True)
```